### DISCUSSION TITLE
What are your dreams/visions going forward?

### DISCUSSION THREAD

**Discussion by CorridorWanterer** [nroutasuo/level13#279] [General]
nroutasuo (I was pondering if I should capitalize your name. I didn't), what are your visions/dreams going forward with this game? I understand that this is a little side project, but what is the goal or what does completion look like in the future?

**Comment by nroutasuo** [comment]
Here's the rough plan at the moment:
* Next version (going live very soon) will introduce a proper ending and wrap up the core mechanics and progression
* Version after that will probably be mostly bug-fixing and technical polish
* Then I will focus a lot more on the story and lore and add a lot more content over a couple of versions
* After that the game is sort of complete
* Then I will take a look at my very long list of random ideas and improvements and see what I still want to add

**Reply by CorridorWanterer** [reply]
Awesome! Sound's like a plan.
Also, I'm sure there are other people who would be sort of willing to help. I don't know anything, but I know that some most definitely have the skills to do so.

**Reply by Turtlecodez** [reply]
I'm excited for the new updates! I've played a lot of different versions of this game, and it's extremely underrated, I love it!
